pubspec.yaml

```
@@ -8,13 +8,40 @@ environment:
   flutter: ^3.32.7
 
 dependencies:
+  collection: ^1.19.1
   flutter:
     sdk: flutter
+  flutter_svg: ^2.2.0
+  yumemi_weather:
+    git:
+      url: https://github.com/yumemi-inc/flutter-training-template.git
+      ref: main
+      path: packages/yumemi_weather
 
 dev_dependencies:
+  build_runner:
+  flutter_gen_runner:
```

[nits]

バージョン指定することが多いので、以下のようにするのが好ましいです🙇‍♂️

```suggestion
  build_runner: ^2.6.0
  flutter_gen_runner: ^5.11.0
```